### ISSUE TITLE
Let's document: Apple iOS, Cisco IOS, and MikroTik RouterOS commands

### ISSUE THREAD

**Issue by reinhart1010** [tldr-pages/tldr#12925]
+ `ios`: To document the Apple iOS and iPadOS. Third-party apps featuring custom command-line experience (such as #12916) should be accepted here as well, though this may subject to change.
+ `cisco-ios`: To document Cisco IOS commands, useful for network administration.
+ `routeros`: To document MikroTik RouterOS commands, useful for network administration.

MikroTik RouterOS features **command hierarchy mechanism**, which allows network administrators to avoid repetitive commands such as:

```
> /ip route print
> /ip route ping 10.0.0.1
```

to become:

```
> /ip route
/ip/route > /print
/ip/route > /ping 10.0.0.1
```

And similarly, Cisco IOS features **modes** which do differ by their (often confusing) command prompts:

```
ROMMON>
Router >
Router #
Router(config)#
Router(config-if)# 
Router(config-subif)
```

I think the documentation hierarchy of Cisco IOS should be slightly changed from `pages.{{lang}}/cisco-ios/{{command}}.md` to `pages.{{lang}}/cisco-ios/{{mode}}/{{command}}.md` (and `pages.{{lang}}/cisco-ios/common/{{command}}.md` for commands shared between modes), since the available set of commands differs between between modes. Citing from [an old Cisco training documentation](https://www.cisco.com/E-Learning/bulk/public/tac/cim/cib/using_cisco_ios_software/02_cisco_ios_hierarchy.htm) for IOS:

> Command options and applications vary, depending on your position within this hierarchy. **Configuration command options are not available until you have navigated to the configuration branch of the Cisco IOS command-line interface (CLI) structure.** When you are in the configuration branch, you may enter system-level configuration commands that apply to the entire router at the global configuration level. Interface-specific configuration commands are available when you switch to the interface configuration level. More detailed information and examples of how to navigate through the Cisco IOS CLI hierarchy are offered in the router configuration labs that follow in a later section of this module.

**Comment by Managor** [comment]
I kinda want router OS's as their own category too. Recently I had to deal with cisco-ios and it was a pain to navigate. tldr pages would help with that

**Comment by kbdharun** [comment]
> I kinda want router OS's as their own category too. Recently I had to deal with cisco-ios and it was a pain to navigate. tldr pages would help with that

Agreed, it would be nice to have a few more directories for stable and popular router platforms.
